apps/web/src/hooks/latte/index.ts

```diff
@@ -272,8 +275,7 @@ export function useLatte() {
         Array.isArray(message.content) &&
         message.content.some(
           (c) =>
-            c.type === ContentType.toolCall &&
-            c.toolName === AGENT_RETURN_TOOL_NAME,
+            c.type === 'tool-call' && c.toolName === AGENT_RETURN_TOOL_NAME,
```

Comment:
Are we still using return tools for agents?

Reply:
no